rma/views/rma_views.xml

```diff
@@ -10,6 +10,7 @@
             <search>
                 <field name="name" />
                 <field name="user_id" />
+                <field name="product_id" />
```

Comment:
Why this change? It's better to inherit the view and add the field if it's necessary

Reply:
The product_id field is defined in the base module, so it makes sense to add it to the base search view

Reply:
what do you think @rousseldenis ?